js/src/helpers/grid-checkbox.helper.js

```diff
@@ -0,0 +1,57 @@
+import $ from 'external/jquery';
+
+export default {
+    /**
+     * Simple helper to help displaying Fomantic-UI checkbox within an atk grid.
+     * The master checkbox in the header of the column enable to toggle all
+     * content checkboxes to check or uncheck. A partially checked master checkbox
+     * is displayed if appopriate.
+     */
+    masterCheckbox: function () {
+        $('.table .master.checkbox').checkbox({
+            // check all children
+            onChecked: function () {
+                const $childCheckbox = $(this).closest('.table').find('.child.checkbox');
+                $childCheckbox.checkbox('check');
+            },
+            // uncheck all children
+            onUnchecked: function () {
+                const $childCheckbox = $(this).closest('.table').find('.child.checkbox');
+                $childCheckbox.checkbox('uncheck');
+            },
+        });
+    },
+
+    childCheckbox: function () {
+        $('.table .child.checkbox').checkbox({
```

Comment:
About 1/2 of the CI Behat jobs fail with:

```
tests-behat/grid.feature:79
```

![image](https://github.com/atk4/ui/assets/2228672/9668682d-2e78-4f76-82b2-cb240a97b291)

I did not experienced such Behat failures on other CI jobs. We run Behat testing very frequently, so it seems related with this PR.

Google Chrome Performance tools in developer tools might help you to understand the issue.

Reply:
The IPP=1000 page load is slow, probably because it checks all checkboxes.
![image](https://github.com/atk4/ui/assets/8080934/dc753325-6bc2-477e-b0e3-f78289c58074)

Reply:
I will try to set `fireOnInit: false`. As of now, it is not possible to have checkboxes preset on init.

Reply:
Looks much better.
![image](https://github.com/atk4/ui/assets/8080934/2b6f8f0f-0d05-49f3-a821-8bca26e09c14)
How fortunate this behat ipp test has been added!

Reply:
Done.

Reply:
> I will try to set `fireOnInit: false`. As of now, it is not possible to have checkboxes preset on init.

I guess the init invoked something in the master checkbox which recomputed something using the whole data, ie. O(n^2) complexity. Or some slowdown due FUI JS? In general, 1k rows is no heavy workload, so it should be fast even with 10k rows :)

Reply:
Yes, each checkbox controls all the other checkboxes to see if the master checkbox has to be changed, which is very inefficient. The default is unchecked for all, and cannot be changed yet so this check has no sense.

Reply:
There is still something wrong with the performance. When I set `ipp` to 500 which lists about 250 countries, checking the master checkbox tooks about 500 ms. That is way to slow than I would expect. I am checking what can be done.